doc/api/deprecations.md

```diff
@@ -3453,7 +3457,7 @@ changes:
     description: Documentation-only deprecation.
 -->
 
-Type: Documentation-only (supports [`--pending-deprecation`][])
+Type: Runtime
```

Comment:
If your only applying to non-node_modules code, isn't this an Application deprecation?